### PR TITLE
Updates and Prep to release 1.0.3

### PR DESCRIPTION
**Pull request by abmusse**
Slight updates to get ready to release 1.0.3

## Changes made

Update README
- Add TOC
- Update examples
- Update Contributing

Add DCO bot config to skip org members

Bump version in package.json

Add CHANGELOG.md

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,33 +1,40 @@
...
 
-**Project Status**: (production-ready as a "technology preview")
+**Project Status**: (production ready as a "technology preview")
 
-The objective of this project is to provide a promise based database connector API for DB2 on IBM i.
+**Objective**: provide a promise based database connector for DB2 on IBM i.
 
 
-This project is a wrapper over the [`idb-connector`](https://bitbucket.org/litmis/nodejs-idb-connector) project but returning promises instead of using callbacks.
+This project is a promise based wrapper over the [`idb-connector`](https://bitbucket.org/litmis/nodejs-idb-connector) project.
```
Comment (kadler) [inline README.md]: Need to adjust URL to point to IBM GitHub

File: README.md [hunk 2]
```diff
@@ -36,84 +43,87 @@ Using Async & Await, to run a select statement & displaying the result set:
...
- }
-}
+const { DBPool } = require('idb-pconnector');
+
+async function poolExample() {
+  const pool = new DBPool({ url: '*LOCAL' });
+  const connection = pool.attach();
+  const statement = connection.getStatement();
+
+  await statement.prepare('CALL QSYS2.TCPIP_INFO()');
```
Comment (kadler) [inline README.md]: QSYS2.TCPIP_INFO is not a stored procedure, but a view: https://www.ibm.com/developerworks/community/wikis/home?lang=en#!/wiki/IBM%20i%20Technology%20Updates/page/QSYS2.TCPIP_INFO%20view/comment/254b522e-f864-4c3e-b0a3-b10a6fecd9f5

File: README.md [hunk 3]
```diff
@@ -36,84 +43,87 @@ Using Async & Await, to run a select statement & displaying the result set:
...
-
- } catch (error){
-   console.log(`Error was: \n\n${error.stack}`);
-   pool.retire(connection);
- }
-}
+const { DBPool } = require('idb-pconnector');
+
+async function poolExample() {
+  const pool = new DBPool({ url: '*LOCAL' });
```
Comment (kadler) [inline README.md]: I understand this is not part of this change, but why does `DBPool` take a "url" parameter?

File: README.md [hunk 4]
```diff
@@ -124,74 +134,66 @@ If you want to bind variables pass an array of values as the second parameter.
...
 
-# **API Documentation**
+# **Documentation**
 
-Please refer to the [documentation](https://github.com/IBM/nodejs-idb-pconnector/blob/master/docs/README.md) for usage of the `idb-pconnector`.
+Please refer to [docs](https://github.com/IBM/nodejs-idb-pconnector/blob/master/docs/README.md) for usage.
 
 # **License**
-MIT. View [LICENSE](https://github.com/IBM/nodejs-idb-pconnector/blob/master/LICENSE)
+MIT - View [LICENSE](https://github.com/IBM/nodejs-idb-pconnector/blob/master/LICENSE)
```
Comment (kadler) [inline README.md]:
```suggestion
[MIT](https://github.com/IBM/nodejs-idb-pconnector/blob/master/LICENSE)
```

File: README.md (hunk 3 above)
Comment (abmusse) [reply README.md]: [link to doc](https://github.com/IBM/nodejs-idb-pconnector/tree/master/docs#constructor-dbpooldatabase--config)

It used to determine the database to connect to.

Looking back 'url' might be the best word to use for that.

File: README.md (hunk 4 above)
Comment (abmusse) [reply README.md]: Thanks will update

File: README.md (hunk 1 above)
Comment (abmusse) [reply README.md]: Good catch thanks!

Comment (markdirish) [reply README.md]: Just some minor grammatical changes throughout this file, like 'promise based' should be 'promise-based.' (Lines 1, 5, 8). Also, should we maybe mention that it enables async/await while also tooting the modern chops of the packages? Something like...

This project is a promised-based wrapper over the [`idb-connector`](https://github.com/IBM/nodejs-idb-pconnector) project that enables the use of modern JavaScript's [async/await](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/async_function) syntax.

Also, why is there a blank line between 13 and 15?

File: README.md (hunk 3 above)
Comment (abmusse) [reply README.md]: url in this case defaults to *LOCAL therefore it can be left out in the example.

`const pool = new DBPool();`

File: README.md (hunk 2 above)
Comment (abmusse) [reply README.md]: Interesting, in ACS it was listed under procedures within the QSYS2 schema

any suggestions for alternative stored procedure to use for the example?

File: README.md (hunk 1 above)
Comment (abmusse) [reply README.md]: updated with requested changes

File: README.md (hunk 2 above)
Comment (abmusse) [reply README.md]: updated with requested changes

File: README.md (hunk 3 above)
Comment (abmusse) [reply README.md]: updated with requested changes

File: README.md (hunk 4 above)
Comment (abmusse) [reply README.md]: updated with requested changes
